layouts/default.vue

```diff
@@ -1,11 +1,26 @@
 <template>
   <div class="default-layout" style="position: relative;">
-    <div
-      id="map"
-      v-mapbox="mapboxOptions"
-      @load-locations="loadLocations"
-      @select-locations="selectLocations"
-    />
+    <no-ssr>
+      <v-mapbox
+        id="map"
+        ref="map"
+        :access-token="mapboxAccessToken"
+        map-style="mapbox://styles/global-data-viewer/cjtss3jfb05w71fmra13u4qqm"
+      >
+        <v-mapbox-navigation-control
+          position="bottom-right"
+        ></v-mapbox-navigation-control>
+        <v-mapbox-layer-clickable
```

Comment:
consider clickable versus raster by something based on existing domains.  For example feature/raster from gis or [sources](https://docs.mapbox.com/mapbox-gl-js/style-spec/#sources) from mapbox. Consider using [mixins](https://vuejs.org/v2/guide/mixins.html) for functionality that exists in multiple layer types.

Reply:
fix in: Feat-GDV-196-split-vector-layers-and-configuration